.gitignore

```diff
@@ -15,3 +15,5 @@
 /.release
 .elixir.plt
 erl_crash.dump
+/lib/*/cover
+/lib/*/_build
```

Comment:
We do have lib/logger/_build when you run "mix test" in the logger project,
for example.

On Friday, 1 April 2016, Aleksei Magusev <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/elixir-lang/elixir/pull/4463#discussion_r58138653>:
>
> > @@ -15,3 +15,5 @@
> >  /.release
> >  .elixir.plt
> >  erl_crash.dump
> > +/lib/*/cover
> > +/lib/*/_build
>
> We don't have _build folders today.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/4463/files/d7a482e120c8ccd201d89261862873844e2ded76#r58138653>
>


-- 

Andrea Leopardi
an.leopardi@gmail.com